### PR TITLE
Check 27 - Last statement is obsolet - clear is okay if it is the parameter in a form

### LINKED ISSUE #308: false positive: last statement is CLEAR
clear is okay if the parameter in a form is CHANGING

### PR DESCRIPTION
**Pull request by pcf0**
It's not the nicest approach i could think of, but at least it fixes the false positive.

fix #308

### CODE REVIEW

File: src/checks/zcl_aoc_check_27.clas.abap [hunk 1]
```diff
@@ -16,8 +16,11 @@ CLASS zcl_aoc_check_27 DEFINITION
     TYPES:
       BEGIN OF ty_stmnt,
         statement TYPE string,
-        row       TYPE stmnt_levl,
-        level     TYPE token_row,
+        row       TYPE token_row,
+        row_to    TYPE token_row,
+        level     TYPE stmnt_levl,
```
Comment (pcf0) [inline src/checks/zcl_aoc_check_27.clas.abap]: `row` and `level` were typed incorrectly, so I've adjusted that

**Comment by pcf0** [comment]
@larshp Because of the failed abaplint: What is the desired line length?

**Comment by larshp** [comment]
configured to 120 characters, see https://github.com/larshp/abapOpenChecks/blob/master/abaplint.json#L15

**Comment by pcf0** [comment]
Thanks, I adjusted the line length.

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**